completions/fish

```diff
@@ -110,17 +110,25 @@ complete -c $progname -n "not $noopt" -l noeditmenu -d 'Do not edit/view PKGBUIL
 complete -c $progname -n "not $noopt" -l noupgrademenu -d 'Do not show the upgrade menu' -f
 
 
-complete -c $progname -n "not $noopt" -l provides -d 'Look for matching provders when searching for packages'
+complete -c $progname -n "not $noopt" -l provides -d 'Look for matching providers when searching for packages'
 complete -c $progname -n "not $noopt" -l noprovides -d 'Just look for packages by pkgname'
 complete -c $progname -n "not $noopt" -l pgpfetch -d 'Prompt to import PGP keys from PKGBUILDs'
 complete -c $progname -n "not $noopt" -l nopgpfetch -d 'Do not prompt to import PGP keys'
 
 # Post V7.887
-complete -c $progname -n "not $noopt" -l useask -d 'Automatically resolce conflicts using pacmans ask flag'
+complete -c $progname -n "not $noopt" -l useask -d 'Automatically resolve conflicts using pacmans ask flag'
 complete -c $progname -n "not $noopt" -l nouseask -d 'Confirm conflicts manually during the install'
 complete -c $progname -n "not $noopt" -l combinedupgrade -d 'Refresh then perform the repo and AUR upgrade together'
 complete -c $progname -n "not $noopt" -l nocombinedupgrade -d 'Perform the repo upgrade and AUR upgrade separately'
 
+#Post V8.976
+complete -c $progname -n "not $noopt" -l nomakepkgconf -d 'Use default makepkg.conf'
+complete -c $progname -n "not $noopt" -l makepkgconf -d 'Use custom makepkg.conf location'
+complete -c $progname -n "not $noopt" -l removemake -d 'Remove make deps after install'
+complete -c $progname -n "not $noopt" -l askremovemake -d 'Ask to remove make deps after install'
+complete -c $progname -n "not $noopt" -l noremovemake -d 'Do not remove make deps after install'
+complete -c $progname -n "not $noopt" -l completioninterval -d 'Refresh interval for completion cache'
```

Comment:
makepkgconf and completioninterval both take an argument. I'm not familiar with fish syntax but should that be handled in any way?

Reply:
I'll add a -xa (exclusive argumment) and -f (no files) to those but I think I'll also need it for one or two more I may have a overlooked